collections/_pages/talks-and-presentations.html

```diff
@@ -0,0 +1,135 @@
+---
+layout: page
+permalink: /talks-and-presentations
+title: Meshery Talks and Presentations
+heading: Meshery Talks and Presentations
+redirect_from: talks-and-presentations
+---
+<style>
+.modal-window {
+  position: fixed;
+  background-color: rgba(255, 255, 255, 0.25);
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 999;
+  visibility: hidden;
+  opacity: 0;
+  pointer-events: none;
+  transition: all 0.3s;
+}
+.modal-window:target {
+  visibility: visible;
+  opacity: 1;
+  pointer-events: auto;
+}
+.modal-window > div {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 2.5em 2em 2em 2em;
+  background: rgba(255, 255, 255);
+  border: 3px solid #00d3a9;
```

Comment:
```suggestion
  border: 3px solid #3c494f;
```
@debo19 could you try using this color and see if it looks better? the keppel green might be a bit too bright imo.